README.md

```diff
@@ -93,7 +93,17 @@ A freestanding output function allows writing a GameVersion to output streams.
  
 
 ### Log.h
-Description coming soon.
+A class to log information during conversion. Everything is logged to log.txt in the directory used to run the program. No configuration or setup is required, just include the header and build the .cpp file, then start logging.
+
+The logger uses C++ stream semantics:  
+```Log(LogLevel::Info) << "Message: " << variable;```
+
+Log level specifies a message at the beginning of the logged line, and can be set to any of the following:  
+ * LogLevel::Error  
+ * LogLevel::Warning  
+ * LogLevel::Info  
+ * LogLevel::Debug  
+ * LogLevel::Progress - this is used to set the progress bar in the Fronter
```

Comment:
Addendums:
- Log will clear log.txt file not on program startup but on first use within a program. As this can lead to situations where Fronter reads old session's log on startup before a converter Logs its first message, clear log.txt manually in main.cpp or just export converter version or a similar log to clear it.
- For uses in programs without Fronter, this won't be necessary.
- LogLevel::Progress can be used for anything, but Fronter expects a string that can be passed to std::stoi() - so only an integer.
- When logging decimals, standard cout modifiers apply, like `Log(LogLevel::Info) << std::fixed << std::setprecision(2) << doubleVariable;`